src/main/java/org/opensearch/flowframework/workflow/WorkflowProcessSorter.java

```diff
@@ -118,20 +120,28 @@ public List<ProcessNode> sortProcessNodes(Workflow workflow, String workflowId)
                 .map(e -> idToNodeMap.get(e.source()))
                 .collect(Collectors.toList());
 
-            TimeValue nodeTimeout = parseTimeout(node);
-            ProcessNode processNode = new ProcessNode(
-                node.id(),
-                step,
-                node.previousNodeInputs(),
-                data,
-                predecessorNodes,
-                threadPool,
-                nodeTimeout
-            );
-            idToNodeMap.put(processNode.id(), processNode);
-            nodes.add(processNode);
-        }
+            try {
```

Comment:
This is a second-level of passing up an IOException reading a json on the classpath. We should handle this exception where it's read rather than passing it up.

Reply:
Address this comment by moving the try catch logic to where we read the json file in the latest revision